test/runner/runner.go

```diff
@@ -165,6 +165,7 @@ func (r *Runner) start() error {
 	router.ServeFiles("/assets/*filepath", http.Dir(args.AssetsDir))
 	router.GET("/cluster/:key/:cluster", r.clusterAPI(r.getCluster))
 	router.POST("/cluster/:key/:cluster", r.clusterAPI(r.addHost))
+	router.POST("/cluster/:key/:cluster/vanilla", r.clusterAPI(r.addVanillaHost))
```

Comment:
This seems more like it should be a parameter or part of the post body to me than a separate endpoint.